src/main/java/core/mate/academy/service/MachineService.java

```diff
@@ -26,13 +27,13 @@ public interface MachineService {
      * @param machines - list of machines to be filled with value
      * @param value    - any object of machine sub class
      */
-    void fill(List<Object> machines, Object value);
+    void fill(List<? super Machine> machines, T value);
```

Comment:
The `fill()` method should ensure that the size of the list remains the same after filling. Make sure that the implementation of this method replaces the old values with the new ones without changing the list size.